mesop/examples/web_component/code_mirror_editor/code_mirror_editor_app.py

```diff
@@ -0,0 +1,74 @@
+import mesop as me
+import mesop.labs as mel
+from mesop.examples.web_component.code_mirror_editor.code_mirror_editor_component import (
+  code_mirror_editor_component,
+)
+
+
+@me.stateclass
+class State:
+  code: str = "# Add your Python code here."
+
+
+@me.page(
+  path="/web_component/code_mirror_editor/code_mirror_editor_app",
+  stylesheets=[
+    "https://cdnjs.cloudflare.com/ajax/libs/codemirror/6.65.7/codemirror.min.css"
+  ],
+  # CAUTION: this disables an important web security feature and
+  # should not be used for most mesop apps.
+  #
+  # Disabling trusted types because plotly uses DomParser#parseFromString
```

Comment:
Just checking, do you still need to disable trusted types? if you do, update comment.

Reply:
Ah good catch. Seems to work fine if I remove that line.